src/validations/commentvalidation.ts

```diff
@@ -0,0 +1,7 @@
+import { string, object } from 'yup';
+
+export default object({
+  body: object({
+    commentBody: string().required().min(5, 'Comment is too short'),
```

Comment:
remove min check, with comment user has the right to add any comment length he wants